cel/canonical_eventlog.go

```diff
@@ -357,12 +334,7 @@ func DecodeToCELR(buf *bytes.Buffer) (r Record, err error) {
 // Replay takes the digests from a Canonical Event Log and carries out the
 // extend sequence for each PCR in the log. It then compares the final digests
 // against a bank of PCR values to see if they match.
```

Comment:
nit: add comments to support RTMR replay as well.

Reply:
done